generated/nidcpower/nidcpower/session.py

```diff
@@ -5156,6 +5201,161 @@ def create_advanced_sequence(self, sequence_name, property_names, set_as_active_
+                specifies channels 0 and 2). A channel range is a lower bound channel followed by a
+                hyphen (-) or colon (:) followed by an upper bound channel (for example, 0-2
+                specifies channels 0, 1, and 2).
+
+                If independent_channels is True and a channels argument is supplied, resource_name
+                will be combined with channels to form a resource string that specifies the
+                instrument and channel(s). This operation is only valid if resource_name refers to a
+                single device. To create a session across channels on multiple devices, do not
+                provide a channels argument and use a resource_name of the form described above.
+
+            reset (bool): If independent_channels is True (default), specifies whether to reset
+                channel(s) during the initialization procedure. Otherwise, specifies whether to
+                reset the device during the initialization procedure.
+
+                See the target initialization method for more details.
+
+            option_string (dict): Specifies the initial value of certain properties for the session.
+                The syntax for optionString is a list of properties with an assigned value where 1
+                is True and 0 is False. For example:
+
+                Simulate=0, DriverSetup=Model:<model number>; BoardType:<bus connector>
+
+                To simulate a multi-instrument session when independent_channels is True, set
+                Simulate to 1 and list multiple instruments for DriverSetup. For example:
+
+                Simulate=1, DriverSetup=ResourceName:<instrument name>; Model:<model number>;
+                BoardType:<bus connector> & ResourceName:<resource name>; Model:<model number>;
+                BoardType:<bus connector>
+
+                You do not have to specify a value for all the properties. If you do not specify a
+                value for a property, the default value is used.
+
+            independent_channels (bool): Specifies whether to initialize the session with
+                independent channels. Set this argument to False on legacy applications or if you
+                are unable to upgrade your NI-DCPower driver runtime.
+
+
+        Returns:
+            vi (int): Returns a session handle that you use to identify the device in all
+                subsequent NI-DCPower method calls.
+
+        '''
+
+        if independent_channels:
+            if channels:
+                import warnings
+                warnings.warn(
+                    "Attempting to initialize an independent channels session with a channels argument. "
+                    "Channels will be combined with the resource name to form a fully-qualified channel list. "
+                    "To avoid this warning, use a fully-qualified channel list as the resource name instead "
+                    "of providing a channels argument.",
```

Comment:
Since we are doing the combining later on, it'd be great to tell the customer what the combined string should/can be in the warning.

Reply:
341b19977b97e27d47d1c0210490fd34d1c7fd80